deepchecks/recommender/checks/label_popularity_drift.py

```diff
@@ -24,7 +24,71 @@
 __all__ = ['LabelPopularityDrift']
 
 class LabelPopularityDrift(TrainTestCheck):
-
+    """
+    Calculate label popularity drift between train dataset and test dataset, using statistical measures.
+
+    Check calculates a drift score for the label in test dataset, by comparing its distribution to the train
+    dataset.
+
+    For numerical columns, we use the Kolmogorov-Smirnov statistic.
```

Comment:
This is copied from some other check - I would guess here we use only EMD, and it's not run on columns